### ISSUE TITLE
Build Form For Accepting Parent Absence Request

### ISSUE THREAD

**Issue by steveshogren** [steveshogren/overseer#41]
Now that we have parent emails, we can built out a way to have the parents email in or something to indicate why a student will be absent. Perhaps a doctors note image or the like.

**Comment by steveshogren** [comment]
I dont have any great ideas at this time how to build this.

**Comment by schmave** [comment]
Whoa, that's a cool idea to receive email on the Attendance Clerk's behalf! Interesting...

**Comment by steveshogren** [comment]
Yeah, I'm thinking something where they go to a page, then put in their email address, it shows them the students associated with that email address, they pick one, it emails them a link, they click that, and then they can input the day and the reason and an attachment. Kinda complicated, but then we don't need authentication.

**Comment by steveshogren** [comment]
Otherwise I'm not sure how to receive email easily, I'll look into that today. Maybe something like you put the student's full name as the subject and the reason?

**Comment by schmave** [comment]
Yeah, if the student's name is in the subject that would work probably. You have the parent's email from the "From" address, so that narrows down the options. It sounds a bit error-prone but kind of fun :)
